### DISCUSSION TITLE
Linking quarto website pages with different parent folders

### DISCUSSION THREAD

**Discussion by GitBixa** [quarto-dev/quarto-cli#6334] [Q&A] [answered]
### Description

I am building a website in quarto and i want to create a page link for PAGE_B.qmd in PAGE_A. qmd.

PAGE_B.qmd is in ROOT_DIRECTORY/FOLDER_B/PAGE_B.qmd whereas PAGE_A.qmd is in ROOT_DIRECTORY/FOLDER_A/PAGE_A.qmd.

In PAGE_A.qmd, i created link to `[page B](FOLDER_B/PAGE_B.qmd)`, however after rendering webpage, it renders `[page B](FOLDER_A/FOLDER_B/PAGE_B.qmd)` which does not exist.

I tried linking using relative path that is `[page B](.../FOLDER_B/PAGE_B.qmd)` but it still renders `[page B](FOLDER_A/.../FOLDER_B/PAGE_B.qmd)` which does not exist

Any solutions to how i can create this linking in quarto is dearly appreciated

**Comment by cscheid** [comment, marked as answer]
> I tried linking using relative path

You have the syntax for relative path wrong: it's `..`. You can also use `/FOLDER_B/PAGE_B.qmd` which is a quarto-specific feature that ensures those paths are always resolved with respect to the website root.

**Reply by GitBixa** [reply]
Thanks so much. It works like a charm
